xio-core/src/main/java/com/xjeffrose/xio/http/internal/Http1Response.java

```diff
@@ -33,7 +33,7 @@ public Http1Response(HttpResponse delegate) {
 
   @Override
   public boolean endOfStream() {
-    return true;
+    return false;
```

Comment:
these xio message classes could use some documentation and/or better names

Reply:
I think we should make some sample/example code to help us:
1) refine the look and feel of the interfaces
2) point at from a user guide
3) we should actually add javadoc too :)